engines/dfc_provider/app/services/affiliate_sales_data_builder.rb

```diff
@@ -0,0 +1,171 @@
+# frozen_string_literal: true
+
+# rubocop:disable Metrics/ClassLength
+class AffiliateSalesDataBuilder < DfcBuilder
+  class << self
+    def build_person(user)
+      DataFoodConsortium::Connector::Person.new(
+        urls.affiliate_sales_data_person_url(user.id),
+        logo: nil,
+        firstName: nil,
+        lastName: nil,
+        affiliatedOrganizations: user_enterprises(user.enterprises)
```

Comment:
No INRAE won't be a manager, otherwise they have access to all the data which we don't want to. We agreed on user flipper to activate their access to this endpoint. copying from the main issue:

> The email address can be authorized against our feature toggles. For example Flipper.enable(:inrae, "jo@inrae.org"). Then we can create a model class to work with Flipper: InraeUser = Struct.new(:flipper_id) and pass that on to check if the email is allowed to access this endpoint.

Reply:
Yes, I have not added it just yet, just wanted to have a first round of comments around the endpoint data. Will add it soon.

Reply:
Hi @anansilva , I've created the method for enterprises to choose to be included, and created a method `User#affiliate_enterprises` that can be used here. See final commit here: https://github.com/openfoodfoundation/openfoodnetwork/pull/12676.

Does this look like what you're expecting? If so, let me know and I'll complete the PR.

Reply:
I'll go ahead with it now and we can adjust later if needed.